experimental/language-generation/csharp_dotnetcore/05.a.multi-turn-prompt-with-language-fallback/Dialogs/UserProfileDialog.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using System.Collections.Generic;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Choices;
+using Microsoft.Bot.Builder.LanguageGeneration;
+using Microsoft.Bot.Builder.LanguageGeneration.Generators;
+using ActivityBuilder = Microsoft.Bot.Builder.Dialogs.Adaptive.Generators.ActivityGenerator;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class UserProfileDialog : ComponentDialog
+    {
+        private readonly IStatePropertyAccessor<UserProfile> _userProfileAccessor;
+        private static MultiLingualTemplateEngineManager _lgGenerator;
+
+        public UserProfileDialog(UserState userState)
+            : base(nameof(UserProfileDialog))
+        {
+            _userProfileAccessor = userState.CreateProperty<UserProfile>("UserProfile");
+            // combine path for cross platform support
+            Dictionary<string, List<string>> lgFilesPerLocale = new Dictionary<string, List<string>>();
```

Comment:
```suggestion
            Dictionary<string, List<string>> lgFilesPerLocale = new Dictionary<string, List<string>>() {
                { "",  new List<string>() { Path.Combine(".", ....)} } 
                { "fr", new List<string>() { Path.Combine(".", ....)} }
            }
```

Reply:
By put it into one constructor, we can show the key-> value mapping more clearly.

Reply:
Fixed.